tools/podlogs

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+function usage {
+    >&2 echo "Usage:"
+    >&2 echo "podlogs [--freshness AGE] [-n numrecords ] <podname>"
+}
+
+if [[ $# -lt 1 ]]
+then
+    usage
+    exit 1
+fi
+
+args=()
+freshness=10d
```

Comment:
I like this default and the ability to change it!

Reply:
Thanks.